src/main/scala/com/codacy/client/bitbucket/v2/service/RepositoryServices.scala

```diff
@@ -103,4 +101,10 @@ class RepositoryServices(client: BitbucketClient) {
         client.executePaginated(Request(baseRequestUrl, classOf[Seq[BranchRestriction]]))
     }
   }
+
+  private def generateRepositoryUrl(owner: String, repo: String): String = {
```

Comment:
Codacy found an issue: [generateRepositoryUrl should have a smaller scope by using private[this]](https://app.codacy.com/gh/codacy/bitbucket-scala-client/pullRequest?prid=5907044)